### PR TITLE
Optimize tiflash-ghpr-unit-tests & tiflash-ghpr-integration-tests

### PR DESCRIPTION
**Pull request by solotzg**
optimize time to checkout 

original: 6m32s
https://ci.pingcap.net/blue/organizations/jenkins/tiflash-ghpr-unit-tests/detail/tiflash-ghpr-unit-tests/5352/pipeline/18

optimized: 13s
https://ci.pingcap.net/blue/organizations/jenkins/tiflash-ghpr-unit-tests/detail/tiflash-ghpr-unit-tests/5371/pipeline/18

----

original: 4m50s
https://ci.pingcap.net/blue/organizations/jenkins/tiflash-ghpr-integration-tests/detail/tiflash-ghpr-integration-tests/8329/pipeline/92

optimized: 33s
https://ci.pingcap.net/blue/organizations/jenkins/tiflash-ghpr-integration-tests/detail/tiflash-ghpr-integration-tests/8337/pipeline/26

### CODE REVIEW

File: jenkins/pipelines/ci/tiflash/tiflash-ghpr-unit-tests.groovy [hunk 1]
```diff
@@ -216,6 +218,8 @@ run_with_pod {
             ])
         }
         dir("/tmp/tiflash-data") {
+            sh "rm -rf /home/jenkins/agent/workspace/tiflash-build-common/tiflash/.git"
+            sh "rm -rf /home/jenkins/agent/workspace/tiflash-build-common/tiflash/contrib"
```
Comment (solotzg) [inline jenkins/pipelines/ci/tiflash/tiflash-ghpr-unit-tests.groovy:222]:
```suggestion
```

**Comment by solotzg** [comment]
cc @purelind

**Review by purelind — APPROVED** [review]
LGTM

**Review by purelind — APPROVED** [review]
LGTM

**Pull request merged**